src/org/jetbrains/kotlin/test/helper/actions/GeneratedTestComboBoxAction.kt

```diff
@@ -128,9 +129,18 @@ class GeneratedTestComboBoxAction(val baseEditor: TextEditor) : ComboBoxAction()
                 .submit(AppExecutorUtil.getAppExecutorService())
         }
 
+        private val VirtualFile.nameWithoutAllExtensions get() = name.split('.').first()
```

Comment:
It looks like this extension can be replaced with `asPathWithoutAllExtensions`.

Reply:
On the second thought, since you consider lack of attention to performance details good manners, then why do you suggest removing this function and using a slower one instead?

Reply:
In theory, the other function needs to deal with file separators on top of just removing the extensions, which could be more work

Reply:
> On the second thought, since you consider lack of attention to performance details good manners, then why do you suggest removing this function and using a potentially slower one instead?

In this case I choose a bit overhead over function duplication.

Reply:
See: [c9c6ef2](https://github.com/demiurg906/test-data-helper-plugin/pull/34/commits/c9c6ef2145f539bd394df60c7bf3db07a78c85b1)